### DISCUSSION TITLE
In VPC CNI EKS add-on, can "rollingUpdate.maxUnavailable" of aws-node DaemonSet be customized?

### DISCUSSION THREAD

**Discussion by grandich** [aws/amazon-vpc-cni-k8s#2830] [General]
We are required to adjust the default value of 10% to a more conservative figure.

Additionally, it's important to note that modifying this parameter alone should not initiate an update of the daemonset.

**Comment by jdn5126** [comment]
@grandich this is configurable via the helm chart: https://github.com/aws/amazon-vpc-cni-k8s/blob/master/charts/aws-vpc-cni/values.yaml#L162. It is not currently configurable via the EKS Managed Addon. If you would like it to be, please file a request here: https://github.com/aws/containers-roadmap/issues
